packages/gestalt/src/Button.js

```diff
@@ -143,6 +155,85 @@ const ButtonWithForwardRef: React$AbstractComponent<
   );
 
   /* eslint-disable react/button-has-type */
+  if (props.type === 'link') {
+    const { href, rel, target } = props;
+
+    return (
+      <Link
+        hoverStyle="none"
+        href={href}
+        ref={innerRef}
+        rel={rel}
+        target={target}
+        inline
+        rounding="pill"
+      >
+        <div
+          style={{ alignItems: 'center', display: 'flex' }}
```

Comment:
@AlbertCarreras let's add it to a CSS file instead of using inline styles. (perf)

Reply:
Ok. Makes sense. It was easy for me to see if I was correcting it.

Reply:
DONE